doc/pages/keys.asciidoc

```diff
@@ -367,6 +327,61 @@ the Shift modifier and moving will extend each selection instead.
 *<a-(>*::
     rotate selections content backward
 
+== Changes through external programs
+
+Shell expansions are available, (See <<expansions#shell-expansions,`:doc expansions shell-expansions`>>)
+The default command comes from the *|* register by default (See <<registers#,`:doc registers`>>)
+
+*|*::
+    pipe each selection through the given external filter program and
+    replace the selection with its output.
+
+*<a-|>*::
+    pipe each selection through the given external filter program and
+    ignore its output.
+
+*!*::
+    insert command output before each selection.
+
+*<a-!>*::
+    append command output after each selection.
+
+== Searching
+
+Searches use the */* register by default (See <<registers#,`:doc registers`>>)
+
+*/*::
+    search (select next match)
+
+*<a-/>*::
+    search (select previous match)
+
+*?*::
+    search (extend to next match)
+
+*<a-?>*::
+    search (extend to previous match)
+
+*n*::
+    select next match
```

Comment:
While you are at it, make it clear that `n` only select the next match from the main selection, while `/` select the next match from each selections.